lib/Doctrine/DBAL/Migrations/Tools/Console/Command/AbstractCommand.php

```diff
@@ -88,7 +89,7 @@ protected function getMigrationConfiguration(InputInterface $input, OutputInterf
     {
         if ( ! $this->migrationConfiguration) {
             if ($this->getHelperSet()->has('configuration')
-                && $this->getHelperSet()->get('configuration') instanceof ConfigurationHelper) {
+                && $this->getHelperSet()->get('configuration') instanceof ConfigurationHelperInterface) {
                 $configHelper = $this->getHelperSet()->get('configuration');
```

Comment:
If the `configuration` helper set exists, but it does *not* implement `ConfigurationHelperInterface`, could we throw an Exception instead? It seems like a misconfiguration - and it would silently fallback to the normal one.

Reply:
As much as I'd love to throw an exception, `configuration` is too generic to assume that there should only be a migrations configuration helper in there. An exception might be overkill, but we should think about how to make this more specific.

Reply:
Hmm, that makes sense, ok :)